### PR TITLE
Style/cocktailcardbutton

### PR DESCRIPTION
**Pull request by Melizzo**
# PR Description
Please include a summary of the change:
added button to cocktail card, and basic styling.

Commit Emojis:
:tada: Initial commit
:art: CSS modifications/updates.

If it is a new feature, describe the feature and the iteration of the project spec it addresses.
Iteration 2 

Note any questions or struggles as well.

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Tests (non-breaking change which adds testing)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the Turing style guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/CocktailCard/CocktailCard.scss [hunk 1]
```diff
@@ -30,3 +34,12 @@
   padding: 0.5% 1%;
   width: 13%;
 }
+
+.cc-button {
```
Comment (asodemann18) [inline src/CocktailCard/CocktailCard.scss:38]: Nice button styling :)

File: src/CocktailCard/CocktailCard.tsx [hunk 2]
```diff
@@ -10,10 +10,13 @@ export interface CocktailsCardProps {
 const CocktailCard: React.SFC<CocktailsCardProps> = (props) => {
   return (
     <section className="cocktail-card">
-      <h3 className="card-title">{props.strDrink}</h3>
-      <img className='CC-img' src={props.strDrinkThumb} alt={props.strDrink} />
+      <div className='card-title'>
+        <h3>{props.strDrink}</h3>
+        <button className="cc-button">Make Me!</button>
```
Comment (asodemann18) [inline src/CocktailCard/CocktailCard.tsx:15]: Love the button name!

**Review by asodemann18 — APPROVED** [review]
Looks great!

**Comment by st0nebreaker** [comment]
Thanks Melissa!

**Comment by st0nebreaker** [comment]
Mel, on CocktailCard.tsx if you import import { Link } from 'react-router-dom'; at the top, then wrap the button in the link as it's done on Dashboard.tsx line 46, <Link to={`/${props.idDrink}/details`}>, it'll be linked up to the dynamic routing to the Cocktail Details page! Alex + I set up the CocktailDetails routing this morning (from App), to dynamically route this if the Link tag is set up :)

**Pull request merged**